ui/pages/ApplicationDetail.tsx

```diff
@@ -69,12 +87,52 @@ function ApplicationDetail({ className, name }: Props) {
 
   const { application = {} } = res;
 
+  //passed into Page as topRight to appear next to breadcrumbs
+  const AppDetailRemoveButton = (
```

Comment:
:thinking:  A bit weird to split this into a variable since it is only used once